File: C#/TwoRockets/TwoRockets/TwoRockets.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TwoRockets
{
    class TwoRockets
    {
        static void Main(string[] args)
        {
            tri();
            plusln();
            rec();
            plusln();
            us();
            plusln();
            rec();
            plusln();
            tri();

            Console.ReadLine();
        }
        public static void tri()
        {
            Console.WriteLine("  / \\      / \\");
            Console.WriteLine(" /   \\    /   \\");
		    Console.WriteLine("/     \\  /     \\");
        }
        public static void plusln()
        {
            Console.WriteLine("+------+ +------+");
        }
        public static void rec()
        {
            Console.WriteLine("|      | |      |");
            Console.WriteLine("|      | |      |");
        }
        public static void us()
        {
            Console.WriteLine("|United| |United|");
            Console.WriteLine("|States| |States|");
        }
    }
}

File: Java/Classwork/src/TwoRockets.java
public class TwoRockets 
{
	public static void main(String[] args) 
	{
		tri();
		plusln();
		rec();
		plusln();
		us();
		plusln();
		rec();
		plusln();
		tri();
	}
	public static void tri()
	{
		System.out.println("  / \\      / \\");
		System.out.println(" /   \\    /   \\");
		System.out.println("/     \\  /     \\");
	}
	public static void plusln()
	{
		System.out.println("+------+ +------+");
	}
	public static void rec()
	{
		System.out.println("|      | |      |");
		System.out.println("|      | |      |");
	}
	public static void us()
	{
		System.out.println("|United| |United|");
		System.out.println("|States| |States|");
	}
}